### PR TITLE
[#2536] Enable xdebug if it's installed but not enabled through php.ini

### LINKED ISSUE #2536: CLI config
Hello,

My composer now reports I should disable my xdebug extension.
I do agree, but I still need it (a lot) for debugging my PHP scripts.
So in order to disable xdebug only in cli mode, I tried to reach the cli's php.ini file... and failed.

I tried to created a php.cli.ini and a cli/php.ini but cli is still using the default php.ini file.

What is the solution to use a separated php.ini file for cli ?

### PR DESCRIPTION
**Pull request by deviantintegral**
Fixes #2536.

Is there a variable I should use instead of /usr/local/opt?

### CODE REVIEW

**Comment by javian** [comment]
I think you should use HOMEBREW_PREFIX (see the Formula cookbook) in case someone is not using the default.

File: Formula/phpunit.rb [hunk 1]
```diff
@@ -17,7 +17,26 @@ class Phpunit < Formula
   def install
     libexec.install "phpunit-#{version}.phar"
     sh = libexec + "phpunit"
-    sh.write("#!/usr/bin/env bash\n\n/usr/bin/env php -d allow_url_fopen=On -d detect_unicode=Off #{libexec}/phpunit-#{version}.phar $*")
+
+    wrapper = <<END
+#!/usr/bin/env bash
+
+PHP_VERSION=$(/usr/bin/env php -r 'echo PHP_MAJOR_VERSION . PHP_MINOR_VERSION;')
+XDEBUG_SO=/usr/local/opt/php$PHP_VERSION-xdebug/xdebug.so
```
Comment (ablyler) [inline Formula/phpunit.rb]: This can be simplified to use something like: `#{Formula['php56-xdebug'].opt_prefix}`

**Comment by ablyler** [comment]
You can shell out to `brew --prefix`to replace `/usr/local`.

**Comment by javian** [comment]
That's exactly what the HOMEBREW_PREFIX contains =)

**Comment by ablyler** [comment]
@deviantintegral can you update your pull request to address the suggestion that @javian left? thanks!

**Comment by javian** [comment]
@deviantintegral ping ^^

**Comment by ablyler** [comment]
@javian do you want to update this for @deviantintegral ?

**Comment by josegonzalez** [comment]
Closing as there hasn't been any update in 2+ weeks. Feel free to re-issue it in a new pull request if you'd like.

**Pull request closed without merging**